### PR TITLE
Enrich agent with all sorts of stuff

### PR DESCRIPTION
**Pull request by underyx**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -17,6 +17,9 @@ inputs:
   publishDeployment:
     description: |
       Deployment ID on semgrep app.
+  generateSarif:
+    description: |
+      Set if you want a semgrep.sarif file to be written with all semgrep findings.
```
Comment (nbrahms) [inline action.yml]: Probably useful to say what to set it to... e.g. `1`, `true`, etc.

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  - repo: https://github.com/psf/black
+    rev: 19.10b0
+    hooks:
+      - id: black
+
+  - repo: https://github.com/asottile/reorder_python_imports
+    rev: v2.3.0
+    hooks:
+      - id: reorder-python-imports
```
Comment (nbrahms) [inline .pre-commit-config.yaml:25]: IMO isort >> reorder-python-imports

**Review by nbrahms — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (underyx) [reply action.yml]: Right. Any of them would work, but I'll say set to 1.

File: .pre-commit-config.yaml (hunk 2 above)
Comment (underyx) [reply .pre-commit-config.yaml:25]: I prefer it too. But don't feel like configuring it now, so just created an issue here: https://github.com/returntocorp/template/issues/7

**Pull request merged**